plugins/pagerduty/README.md

```diff
@@ -39,12 +39,26 @@ import {
 {
   isPagerDutyAvailable(entity) && (
     <Grid item md={6}>
-      <PagerDutyCard entity={entity} />
+      <EntityProvider entity={entity}>
+        <PagerDutyCard />
+      </EntityProvider>
```

Comment:
Actually, the right fix at this point is to NOT use `EntityProvider` at all, but to instead render `<EntityPagerDutyCard />` which is also exported by the same package. That assumes that the application is migrated to use the new composability stuff, which should be the case now for end users.